pkg/xen-tools/patches-4.19.0-rc4-v3/x86_64/08-Revert__Revert__vfio_pci-quirks_c__Disable_stolen_memory_for_igd_VFIO__.patch

```diff
@@ -0,0 +1,94 @@
+diff --git a/tools/qemu-xen/hw/vfio/igd.c b/tools/qemu-xen/hw/vfio/igd.c
+index b31ee79..e85d26a 100644
+--- a/tools/qemu-xen/hw/vfio/igd.c
++++ b/tools/qemu-xen/hw/vfio/igd.c
+@@ -252,7 +252,6 @@ static int vfio_igd_gtt_max(VFIOPCIDevice *vdev)
+     uint32_t gmch = vfio_pci_read_config(&vdev->pdev, IGD_GMCH, sizeof(gmch));
+     int ggms, gen = igd_gen(vdev);
+ 
+-    gmch = vfio_pci_read_config(&vdev->pdev, IGD_GMCH, sizeof(gmch));
+     ggms = (gmch >> (gen < 8 ? 8 : 6)) & 0x3;
+     if (gen > 6) {
+         ggms = 1 << ggms;
+@@ -378,16 +377,11 @@ void vfio_probe_igd_bar4_quirk(VFIOPCIDevice *vdev, int nr)
+     uint16_t cmd_orig, cmd;
+     Error *err = NULL;
+ 
+-    /*
+-     * This must be an Intel VGA device at address 00:02.0 for us to even
+-     * consider enabling legacy mode.  The vBIOS has dependencies on the
+-     * PCI bus address.
+-     */
++   
+     if (!vfio_pci_is(vdev, PCI_VENDOR_ID_INTEL, PCI_ANY_ID) ||
+-        !vfio_is_vga(vdev) || nr != 4 ||
+-        &vdev->pdev != pci_find_device(pci_device_root_bus(&vdev->pdev),
```

Comment:
I'm wondering what was the original reason of this patch and if we still need it.... @rucoder , do you know if we still need to revert these changes from Xen?

Reply:
@rene As of now, I don't see any issue with my updated revert vfio patch so keeping the updated one.